test/pg.ddl.unit.js

```diff
@@ -0,0 +1,788 @@
+'use strict';
+
+const { join } = require('path');
+const common = require('@metarhia/common');
+const metatests = require('metatests');
+const metaschema = require('metaschema');
+const ddl = require('../lib/pg.ddl');
+
+const schemasDir = join(__dirname, 'fixtures/ddl-unit');
+const unitTests = metatests.test('pg.ddl unit tests');
+metaschema.fs.loadAndCreate(schemasDir, { common }, (err, ms) => {
+  const {
+    decorators: {
+      Index: createIndex,
+      Unique: createUnique,
+      Include: createInclude,
+    },
+  } = metaschema;
+
+  unitTests.error(err);
+  unitTests.endAfterSubtests();
+
+  unitTests.testSync('pg.ddl pad', test =>
+    test.strictSame(ddl.pad('__NAME__', 18, '#'), '__NAME__##########')
+  );
+
+  unitTests.testSync('pg.ddl padProperty', test =>
+    test.strictSame(ddl.padProperty('__NAME__', 18), '"__NAME__"          ')
+  );
+
```

Comment:
You should also add tests, that will ensure function throws an error on invalid args: `padPeoperty('__Name__', 4)`.